frontend/packages/topology/src/behavior/withCreateConnector.tsx

```diff
@@ -162,22 +164,25 @@ const CreateConnectorWidget: React.FC<CreateConnectorWidgetProps> = observer((pr
       {prompt && (
         <ContextMenu
           reference={{ x: prompt.event.pageX, y: prompt.event.pageY }}
+          className={contextMenuClass}
           open
           onRequestClose={() => {
             setActive(false);
             onKeepAlive(false);
           }}
         >
-          {prompt.choices.map((c) => (
-            <ContextMenuItem
-              key={c.label}
-              onClick={() => {
-                onCreate(prompt.element, prompt.target, prompt.event, c);
-              }}
-            >
-              {c.label}
-            </ContextMenuItem>
-          ))}
+          {React.isValidElement(prompt.choices?.[0])
```

Comment:
```suggestion
          {React.isValidElement(prompt.choices)
```

Reply:
Done

Reply:
this change causes `key` error

Reply:
Sorry I forgot to remove this comment of mine .